packages/calendar/test/todolist-1.spec.js

```diff
@@ -0,0 +1,221 @@
+function getPossibleExpressions(expressionType, count) {
+    const data = require('../data/expressions/en.json')
+    switch (expressionType) 
+    { 
+        case "create list":
+            return data.todolist.create_list.expressions
+        case "view lists":
+            return data.todolist.view_lists.expressions
+        case "view list":
+            return data.todolist.view_list.expressions
+        case "rename list":
+            return data.todolist.view_list.expressions
+        case "delete list":
+            return data.todolist.delete_list.expressions
+        case "add todos":
+            return data.todolist.add_todos.expressions
+        case "complete todos":
+            return data.todolist.complete_todos.expressions
+        case "uncheck todos":
+            return data.todolist.uncheck_todos.expressions
+        default:
+            console.log("invalid expression type requested")
+            break
+    }
+}
+
+function getOutputs(){
+    const fs = require('fs')
+    const path = require('path')
```

Comment:
Instead to use the CJS `require` syntax, use the ESM import at the top of the file.